.github/workflows/release-reproducible.yml

```diff
@@ -0,0 +1,142 @@
+name: release-reproducible
```

Comment:
Can we make reproducible builds a default for Lighthouse? Not an additional target requiring extra effort to support